.env.example

```diff
@@ -59,6 +59,8 @@ APP_DEFAULT_LOCALE=en
 # Ability to disable signups on your instance.
 # Can be true or false. Default to false.
 APP_DISABLE_SIGNUP=true
+# Enable user email verification.
```

Comment:
It's more elegant if you put an empty line above it.

Reply:
OK